charts/super-agent-deployment/templates/configmap-fleet-configs.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.config.fleet -}}
+{{- range $fleetName, $fleetConfig := .Values.config.fleet -}}
```

Comment:
why creating N configMaps instead of just one with the folder structure inside? I think it is possible since we need just one level

Reply:
(no strong opinion just curious why you went for that)

Reply:
I thought we did not have to specify the `items:`, I forgot that there were two levels of subfolders from `/etc/newrelic-super-agent/fleet/agents.d` and not just one.

Then I agree, one cm makes sense!